python/server/qgsserverprojectutils.sip

```diff
@@ -243,5 +243,5 @@ namespace QgsServerProjectUtils
       * @param project the QGIS project
       * @return the Layer ids list.
       */
-    QStringList wcsLayers( const QgsProject &project );
+    QStringList wcsLayerIds( const QgsProject &project );
```

Comment:
Just looking - every call to this is followed up by retrieving the layer from the project. Why not change this method to return a QList< QgsMapLayer *> instead of the layer ids.